lib/Doctrine/Common/EventManager.php

```diff
@@ -68,7 +70,7 @@ public function dispatchEvent($eventName, EventArgs $eventArgs = null)
      *
      * @return array The event listeners for the specified event, or all event listeners.
      */
-    public function getListeners($event = null)
+    public function getListeners(?string $event = null): array
```

Comment:
No. `null` is a default value, and strictly speaking, it's an invalid default value for the type. These are valid:
1. `string $event` => must be string, must be given in function call
2. `string $event = ''` => must be string, can be omitted in which case an empty string is used
3. `?string $event` => must be string or null, must be given in function call
4. `?string $event = null` => must be string or null, can be omitted in which case `null` is used

`string $event = null` is invalid, as you'd expect a `string` but receive `null`.